chirp/drivers/radtel_rt620.py

```diff
@@ -0,0 +1,545 @@
+        rs = RadioSetting("vfo.a.wide", "Bandwidth",
+                          RadioSettingValueList(self.MODES,
+                                                current_index=val))
+        vfoA.append(rs)
+        if _mem.vfo.b.wide >= len(self.MODES):
+            val = 0
+        else:
+            val = _mem.vfo.b.wide
+        rs = RadioSetting("vfo.b.wide", "Bandwidth",
+                          RadioSettingValueList(self.MODES,
+                                                current_index=val))
+        vfoB.append(rs)
+        if _mem.vfo.a.scode >= len(self.SCODE_LIST):
+            val = 0
+        else:
+            val = _mem.vfo.a.scode
+        rs = RadioSetting("vfo.a.scode", "Signal Code",
+                          RadioSettingValueList(self.SCODE_LIST,
+                                                current_index=val))
+        vfoA.append(rs)
+        if _mem.vfo.b.scode >= len(self.SCODE_LIST):
+            val = 0
+        else:
+            val = _mem.vfo.b.scode
+        rs = RadioSetting("vfo.b.scode", "Signal Code",
+                          RadioSettingValueList(self.SCODE_LIST,
+                                                current_index=val))
+        vfoB.append(rs)
+        if _mem.vfo.a.step >= len(self.STEPS):
+            val = 0
+        else:
+            val = _mem.vfo.a.step
+        rs = RadioSetting("vfo.a.step", "Tuning Step",
+                          RadioSettingValueList(baofeng_uv17Pro.LIST_STEPS,
+                                                current_index=val))
+        vfoA.append(rs)
+        if _mem.vfo.b.step >= len(self.STEPS):
+            val = 0
+        else:
+            val = _mem.vfo.b.step
+        rs = RadioSetting("vfo.b.step", "Tuning Step",
+                          RadioSettingValueList(baofeng_uv17Pro.LIST_STEPS,
+                                                current_index=val))
+        vfoB.append(rs)
+        rs = RadioSetting("vfo.a.bcl", "Busy Channel Lock",
+                          RadioSettingValueBoolean(_mem.vfo.a.bcl))
+        vfoA.append(rs)
+        rs = RadioSetting("vfo.b.bcl", "Busy Channel Lock",
+                          RadioSettingValueBoolean(_mem.vfo.b.bcl))
+        vfoB.append(rs)
+        rs = RadioSetting("vfo.a.compander", "Compander",
+                          RadioSettingValueBoolean(_mem.vfo.a.compander))
+        vfoA.append(rs)
+        rs = RadioSetting("vfo.b.compander", "Compander",
+                          RadioSettingValueBoolean(_mem.vfo.b.compander))
+        vfoB.append(rs)
+        workmode.append(vfoA)
+        workmode.append(vfoB)
+
+    def _get_raw_memory(self, number):
+        if number == -1:
+            return self._memobj.vfo.b
+        elif number == -2:
+            return self._memobj.vfo.a
+        else:
+            return super()._get_raw_memory(number)
+
+    def get_memory(self, number):
+        #        if isinstance(number, str):
+        #            number = self._get_specials()[number]
```

Comment:
Please remove all this commented-out code for the final version. Without syntax highlighting it can be very difficult to distinguish commented-out code from actual code.

Reply:
These were relicts when I was trying to put VFO1 and VFO2 into channel list but when I did not succeed I have left it there if somebody wanted to continue. It will be removed.

Reply:
Thanks. You're right to want to put those as special channels instead of the "work mode" approach that so many drivers have taken. I might take a shot at fixing that in the base UV-17 driver and maybe I'll tag you to check my work applies for your driver as well, if you're willing.

Reply:
I am willing, there is probably no urgency to have this driver merged right now, better to do everything right and the amount of code here could be significantly reduced (whole `get_settings_workmode` could be removed in that case). 

Only I now remember that when I was experimenting with it it seemed the TX and RX frequency was stored differently in VFOs, but that may be actually just two ways of storing them which are interchangeable.

Reply:
Up to you, I'd rather just get it in and make incremental changes. We can also just remove the workmode stuff for this merge and then follow up. The fact that they're stored differently for the VFOs isn't a problem, just needs a slightly different approach for `get_memory()`

Reply:
No, not preferred at all. The drivers I've written do it the proper way (or exclude the VFOs because I see very little point in letting those be programmed in advance by PC). I think the OEM software for a lot of these chinese radios implement them as separate settings and that pattern has worked its way into CHIRP, even though it's less ideal.

No rush, thanks very much for your work on this. I will plan to push the pending changes tomorrow for a build on Friday. If we can include this by then, then it will be in that release, but if not, the following week.